Cargo.toml

```diff
@@ -32,6 +33,7 @@ serde = { version = "1.0", features = ["derive"], optional = true }
 # of dev-dependencies because of
 # https://github.com/rust-lang/cargo/issues/1596
 serde_json = { version = "1.0", optional = true }
+sqlparser_derive = { version = "0.1", path = "derive", optional = true }
```

Comment:
Note to myself: if we merge this PR I think we should document this feature flag -- it seems documentation is missing for the existing feature flags

Reply:
Docs in https://github.com/sqlparser-rs/sqlparser-rs/pull/774